econet_hvac_furnace.yaml

```diff
@@ -91,7 +93,7 @@ sensor:
     state_class: "measurement"
     entity_category: "diagnostic"
     filters:
-      - delta: 0.9
+      - delta: 0.91
```

Comment:
What's the reason for this change? Was it causing some problem?

Reply:
Digging in a little bit more, there's a default precision in the econet sensor here (0 decimals) that makes my change unnecessary.    For the template sensors the default precision is [1 decimal ](https://github.com/esphome/esphome/blob/c35a21773e3158484c60879a8fdcb806b9cc1163/esphome/components/template/sensor/__init__.py#L19), which led to some odd behaviors. I'll set it back to 0.9 and make the accuracy_decimals 0 for the templates.

```
[12:39:52][V][sensor:043]: 'Supply Air Temperature': Received new state 105.300003
[12:39:52][D][sensor:094]: 'Supply Air Temperature': Sending state 105.30000 °F with 0 decimals of accuracy
```

```
[12:39:55][V][sensor:043]: 'Supply Air Temperature': Received new state 106.099998
[12:39:55][D][sensor:094]: 'Supply Air Temperature': Sending state 106.10000 °F with 1 decimals of accuracy
```